packages/cookie-consent-react/package.json

```diff
@@ -33,12 +33,13 @@
         "dev": "run-p dev:*"
     },
     "dependencies": {
-        "@babel/runtime": "^7.16.0",
+        "@babel/runtime": "^7.16.3",
         "@fremtind/jkl-button-react": "^8.0.4",
         "@fremtind/jkl-cookie-consent": "^5.0.2",
         "@fremtind/jkl-list-react": "^6.0.3",
         "@fremtind/jkl-react-hooks": "^6.0.2",
-        "classnames": "^2.2.6"
+        "classnames": "^2.2.6",
+        "react-hook-form": "^7.20.1"
```

Comment:
@piofinn jeg husker vi hadde en diskusjon om `react-hook-form` for Feedback hvor du gjorde en innsats for å få det til å funke uten. Vi har hatt en implisitt dependency på `portal` sin `react-hook-form` [fra denne pakka](https://github.com/fremtind/jokul/blob/2f948a8ed391434c2de944fbbb7a4eca4de8b152/packages/cookie-consent-react/src/CookieConsentModal.tsx#L7) og shipper hele greia nå i cookie consent på `main`. Gjør quickfixen ved å gjøre det eksplisitt her nå, men anser vi dette som en bug?

Reply:
Hmm.. Er litt usikker. Blir jo litt ekstra pakkestørrelse på denne måten. Kanskje man kan ta en liten sjekk med teamene om folk er over på v7? I så fall kan man jo sette det som peerDep. Kan stå som det er her inntil videre i hvert fall 😊